SqlDataCatalog-Collibra/convert-to-collibra.ps1

```diff
@@ -0,0 +1,203 @@
+
+function Convert-CollibraJSON {
+    param (
+        $communityName = "SqlDataCatalog_ImportedDatabase_11234",
+        [Parameter(Mandatory)][string] $instanceName,
+        [Parameter(Mandatory)][string] $databaseName,
+        [object[]] $columns
+    )
+
+    $export = @()
+
+    # Constants
+
+    $personallyIdentifiableInformationTags = @(
+        "Confidential_-_GDPR",
+        "Highly_Confidential_-_GDPR"
+    )
+
+    $securityClassificationMappings = @{
+        "Confidential" = "Confidential";
+        "Confidential_-_GDPR" = "Confidential";
+        "Highly_Confidential" = "Highly Confidential";
+        "Highly_Confidential_-_GDPR" = "Highly Confidential";
+        "General" = "Internal Only";
+        "Public" = "Public";
+    }
+
+
+    # Community
+
+    $communityIdentifier = [pscustomobject]@{
+        name = $communityName
+    }
+
+    $export += [pscustomobject]@{
+        resourceType = "Community";
+        identifier   = $communityIdentifier;
+        name         = $communityName
+    }
+    # Domains
+
+    $systemsAndDatabasesIdentifier = [pscustomobject]@{
+        name      = "Systems & Databases";
+        community = $communityIdentifier
+    }
+
+    $physicalModelIdentifier = [pscustomobject]@{
+        name      = "Physical Model";
+        community = $communityIdentifier
+    }
+
+    $export += [pscustomobject]@{
+        resourceType = "Domain";
+        identifier   = $systemsAndDatabasesIdentifier;
+        type         = [pscustomobject]@{
+            name = "Technology Asset Domain"
+        }
+    }
+
+    $export += [pscustomobject]@{
+        resourceType = "Domain";
+        identifier   = $physicalModelIdentifier;
+        type         = [pscustomobject]@{
+            name = "Physical Data Dictionary"
+        }
+    }
+
+    # Instance and database
+
+    $acceptedStatus = [pscustomobject]@{
+        name = "Accepted"
+    }
+    $instanceIdentifier = [pscustomobject]@{
+        name   = $instanceName;
+        domain = $systemsAndDatabasesIdentifier
+    }
+    $export += [pscustomobject]@{
+        resourceType = "Asset";
+        identifier   = $instanceIdentifier;
+        displayName  = $instanceName;
+        type         = [pscustomobject]@{
+            name = "Server"
+        };
+        status       = $acceptedStatus;
+    }
+
+    $databaseIdentifier = [pscustomobject]@{
+        name   = "$instanceName>$databaseName";
+        domain = $systemsAndDatabasesIdentifier
+    };
+    $export += [pscustomobject]@{
+        resourceType = "Asset";
+        identifier   = $databaseIdentifier;
+        displayName  = $databaseName;
+        type         = [pscustomobject]@{
+            name = "Database"
+        };
+        status       = $acceptedStatus;
+        relations    = [pscustomobject]@{
+            "00000000-0000-0000-0000-000000007054:SOURCE" = @($instanceIdentifier)
```

Comment:
Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?

Reply:
This appears to be a Collibra constant. Unlike many of the other constants, we couldn't find a human-readable string to replace the GUID.

This goes for the other comments on this topic too.